migrations/20191126194405_add_data_column_user_campaigns.js

```diff
@@ -1,7 +1,9 @@
 exports.up = function (knex, Promise) {
-  return knex.schema.table('users_campaigns', function (t) {
-    t.jsonb('data').default({})
-  })
+  return knex.schema
+    .table('users_campaigns', function (t) {
+      t.jsonb('data').default({})
+    })
+    .raw("CREATE INDEX on ?? ((??#>>'{motive}'))", ['users_campaigns', 'data'])
```

Comment:
mmm does this mean every campaign will need to have this field before saving? Can we do it without this index?

Reply:
Mmm no, why do you think that?

Reply:
No, I'm just asking. I don't think this is necessary since we won't be looking up for users with their motive in the app. We will do it in Metabase for sure but those queries can be slow and will be heavy cached.

But I'm fine leaving it here if it doesn't cause any problems.